documentation/sphinx/source/release-notes.rst

```diff
@@ -42,10 +54,22 @@ Fixes
 * In some cases, calling ``OnError`` with a non-retryable error would partially reset a transaction. As of API version 610, the transaction will no longer be reset in these cases and will instead put the transaction into an error state. `(PR #1298) <https://github.com/apple/foundationdb/pull/1298>`_
 * Standardized datetime string format across all backup and restore command options and outputs. `(PR #1248) <https://github.com/apple/foundationdb/pull/1248>`_
 * Read workload status metrics would disappear when a storage server was missing. `(PR #1348) <https://github.com/apple/foundationdb/pull/1348>`_
+* The ``coordinators auto`` command could recruit multiple coordinators with the same zone ID. `(Issue #988) <https://github.com/apple/foundationdb/issues/988>`_
+* The version of a cluster after a restore could have been lower than the restore version, making version stamp operations get smaller. `(PR #1213) <https://github.com/apple/foundationdb/pull/1213>`_
+* If a process changed its process class while being rebooted, it could recruited for the cluster controller using its previous process class. `(PR #1350) <https://github.com/apple/foundationdb/pull/1350>`_
+* Fixed a few thread safety issues. `(PR #1085) <https://github.com/apple/foundationdb/pull/1085>`_
```

Comment:
Ah, this is the one you were talking about. These were found by valgrind, but I don't know of any issues we've seen because of them. I think both are related to using slow task profiling which is off by default on the client and might not have been possible to hit in the server with the way it's being used now. It probably doesn't hurt to have it in, though, so maybe something like:

```suggestion
* Fixed a few thread safety issues with slow task profiling. `(PR #1085) <https://github.com/apple/foundationdb/pull/1085>`_
```